build/configs/artik053/STDK/Make.defs

```diff
@@ -120,7 +120,7 @@ ARCHDEFINES =
 ARCHPICFLAGS = -fpic -msingle-pic-base -mpic-register=r10
 
 CFLAGS = $(ARCHCFLAGS) $(ARCHWARNINGS) $(ARCHOPTIMIZATION) $(ARCHCPUFLAGS) $(ARCHINCLUDES) $(ARCHDEFINES) $(EXTRADEFINES) -pipe -ffunction-sections -fdata-sections
-CFLAGS += -DMBEDTLS_USER_CONFIG_FILE=\"$(TOPDIR)/../external/stdk/tizenrt-wrapper/mbedtls_config.h\"
+CFLAGS += -DMBEDTLS_USER_CONFIG_FILE=CONFIG_MBEDTLS_USER_FILE
```

Comment:
This is used only in mbedtls. Could you let me know why you added it in top of Make.defs?

Reply:
because it's an customized config for stdk module, so I think this compling variable definition should be put in the build config for stdk. can you share your opinion? thanks

Reply:
and because of it's a customized config for stdk, I thought it could be added in makefile of external/stdk , or in this top of Make.defs,  and as you known this config.h is include in mbedtls and openssl, so there will be error if add it in external/stdk, then I put it here in this Make.defs

Reply:
It's just preset configuration like example. To use stdk, configuring `STDK` is not mandatory. We should make it enable to set from another defconfig. In architecture point of view, related definition or module should be located in its module. This CFLAGS should be in mbedtls.